fgcm/fgcmConfig.py

```diff
@@ -143,6 +140,8 @@ class FgcmConfig(object):
     cycleNumber = ConfigField(int, default=0)
     outfileBase = ConfigField(str, required=True)
     maxIter = ConfigField(int, default=50)
+    deltaMagBkgOffsetPercentile = ConfigField(float, default=0.25)
+    deltaMagBkgPerCcd = ConfigField(bool, default=True)
```

Comment:
The default here is inconsistent with that in `python/lsst/fgcmcal/fgcmFitCycle.py` (not sure if that matters...).